cluster/images/kubemark/Makefile

```diff
@@ -22,15 +22,15 @@
 REGISTRY := $(if $(REGISTRY),$(REGISTRY),staging-k8s.gcr.io)
 IMAGE_TAG := $(if $(IMAGE_TAG),$(IMAGE_TAG),latest)
 
-all: gcloudpush
+all: dockerpush
```

Comment:
? why this change

Reply:
because we're going to use this as well so I find it confusing. I can revert to `gcloudpush`

Reply:
oh, i see how this works. makes sense